lib/pacemaker/pcmk_trans_unpack.c

```diff
@@ -17,6 +17,37 @@
 #include <crm/common/xml.h>
 #include <pacemaker-internal.h>
 
+/*
+ * pcmk_monitor_timeout overrides CRM_meta_timeout for start and monitor
+ * for stonith devices.
+ */
+static void
+unpack_st_mon_timeout(crm_action_t * action)
+{
+    xmlNode *rsc = find_xml_node(action->xml, XML_CIB_TAG_RESOURCE, FALSE);
+    xmlNode *attributes = find_xml_node(action->xml, XML_TAG_ATTRS, FALSE);
+
+    const char *task = crm_element_value(action->xml, XML_LRM_ATTR_TASK);
+    const char *class = NULL;
+    const char *st_mon_timeout = NULL;
+
+    if (rsc)
+        class = crm_element_value(rsc, XML_AGENT_ATTR_CLASS);
```

Comment:
We like to always use braces for the bodies of if/else blocks.  There's a couple places here that could use it.

Reply:
Thanks! Duly noted.